README_EN.md

```diff
@@ -2,6 +2,219 @@
 
 A short tutorial about updating `CentOS 8` / `CentOS Stream 8` to `CentOS Stream 9`
 
-[中文](README.md) | English(WIP)
+[中文](README.md) | [English](README_EN.md)
 
 ---
+
+Listen, dude! Did you know? In December 31st, 2021, `CentOS 8` ~~was finished! 🎉🎉🎉~~
+Are you still using `CentOS 8`? Why not update to `CentOS Stream 8`, `Rocky Linux 8` or `AlmaLinux` ?
+
+Wait, don't you hear that `CentOS Stream 9` has already released? There are even `Nodejs16` , `Golang1.17.5`, `kernel5.14` and `glic 2.34` package indexes available!
+
+If you pursue new features and a `dnf` package manager, `CentOS Stream 9` is probably your second best choice! (The first one is still `Fedora`)
+
+Unfortunately my VPS provider doesn't provide `CentOS Stream 9` image, or even `CentOS Stream 8` image, so I have to use the outdated `CentOS 8` image, isn't that bad?  
+
+Don't worry, with the method provided in this article, you can also upgrade to `CentOS Stream 9` on `CentOS 8`, with all your data kept and no need to reinstall system!

```

Comment:
Don't worry, with the method provided in this article, you can also upgrade to `CentOS Stream 9` from `CentOS 8`, with all your data kept and not have to reinstall your system!